Objective-C/TOCropViewController/Views/TOCropView.m

```diff
@@ -327,7 +327,7 @@ - (void)layoutInitialImage
     if (frame.size.width < scaledSize.width - FLT_EPSILON || frame.size.height < scaledSize.height - FLT_EPSILON) {
         CGPoint offset = CGPointZero;
         offset.x = -floorf((CGRectGetWidth(self.scrollView.frame) - scaledSize.width) * 0.5f);
-        offset.y = -floorf((CGRectGetHeight(self.scrollView.frame) - scaledSize.height) * 0.5f);
+        offset.y = -floorf((CGRectGetHeight(self.scrollView.frame) - scaledSize.height + 68) * 0.5f);
```

Comment:
why 68?

Reply:
Crop was designed for modal presentation, with nav bar logic a bit broken, it's only solution that I found, If it's really mater I can find another solution but it can take too much time?